<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>511</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
   </font>
  </property>
  <layout class="QVBoxLayout" name="mainLayout" stretch="0,1">
   <item>
    <layout class="QHBoxLayout" name="srsOpLayout">
     <item>
      <widget class="QPushButton" name="loadMlr">
       <property name="font">
        <font>
         <family>Consolas</family>
        </font>
       </property>
       <property name="text">
        <string>Load MLR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="createMlr">
       <property name="font">
        <font>
         <family>Consolas</family>
        </font>
       </property>
       <property name="text">
        <string>Create MLR</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveMlr">
       <property name="font">
        <font>
         <family>Consolas</family>
        </font>
       </property>
       <property name="text">
        <string>Save MLR</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="bottomLayout">
     <item row="0" column="0" rowspan="3">
      <widget class="QGroupBox" name="itemsGbx">
       <property name="title">
        <string>Items:</string>
       </property>
       <layout class="QVBoxLayout" name="itemsLayout">
        <item>
         <widget class="QListWidget" name="itemsList">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="chainCfgGbx">
       <property name="title">
        <string>Chain Config:</string>
       </property>
       <layout class="QGridLayout" name="chainLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="chainTypeLbl">
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="text">
           <string>Type:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="chainIdxLbl">
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="text">
           <string>Index:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="chainIdx">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="chainCountLbl">
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="text">
           <string>Count:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="chainCount">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="chainType">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Magic</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Level</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="magicCfgGbx">
       <property name="title">
        <string>Magic Config:</string>
       </property>
       <layout class="QGridLayout" name="magicLayout">
        <item row="4" column="1">
         <widget class="QSpinBox" name="magicIndex">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="magicIndexLbl">
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="text">
           <string>Index:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="magicTypeLbl">
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="text">
           <string>Type:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="magicType">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Attack</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Enhance</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Recovery</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Assist</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Special</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QGroupBox" name="levelCfgGbx">
       <property name="title">
        <string>Level Config:</string>
       </property>
       <layout class="QGridLayout" name="levelLayout">
        <item row="5" column="0">
         <widget class="QLabel" name="bodyMovementLbl">
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="text">
           <string>Body Movement:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="hpMaxLbl">
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="text">
           <string>HP Max:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="defensePower">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="attackPowerLbl">
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="text">
           <string>Attack Power:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="attackPower">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>9999</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="expLevelUpLbl">
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="text">
           <string>Exp Level Up:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="mpMaxLbl">
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="text">
           <string>MP Max:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="mpMax">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="defensePowerLbl">
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="text">
           <string>Defense Power:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="expLevelUp">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="bodyMovement">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>127</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QSpinBox" name="luckyValue">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>127</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="learnedLbl">
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="text">
           <string>Learned:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QSpinBox" name="learned">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QSpinBox" name="spiritPower">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>127</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="spiritPowerLbl">
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="text">
           <string>Spirit Power:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="luckyValueLbl">
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="text">
           <string>Lucky Value:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="hpMax">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="buttonSymbols">
           <enum>QAbstractSpinBox::NoButtons</enum>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>999</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>loadMlr</tabstop>
  <tabstop>createMlr</tabstop>
  <tabstop>saveMlr</tabstop>
  <tabstop>itemsList</tabstop>
  <tabstop>chainType</tabstop>
  <tabstop>chainIdx</tabstop>
  <tabstop>chainCount</tabstop>
  <tabstop>magicType</tabstop>
  <tabstop>magicIndex</tabstop>
  <tabstop>hpMax</tabstop>
  <tabstop>mpMax</tabstop>
  <tabstop>attackPower</tabstop>
  <tabstop>defensePower</tabstop>
  <tabstop>expLevelUp</tabstop>
  <tabstop>bodyMovement</tabstop>
  <tabstop>spiritPower</tabstop>
  <tabstop>luckyValue</tabstop>
  <tabstop>learned</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
